src/utils.ts

```diff
@@ -451,9 +452,30 @@ export function makeProfileJWT(profileData: Object, privateKey: string) : string
   const signedToken = blockstack.signProfileToken(profileData, privateKey);
   const wrappedToken = blockstack.wrapProfileToken(signedToken);
   const tokenRecords = [wrappedToken];
-  return JSONStringify(tokenRecords);
+  return JSON.stringify(tokenRecords);
 }
 
+export async function makeDIDConfiguration(network:CLINetworkAdapter, blockstackID: string, domain: String, privateKey:string): Promise<{did:string, jwt:string}> {
+
+  const tokenSigner = new TokenSigner("ES256K", privateKey)
+  const nameInfo = await network.getNameInfo(blockstackID);
+  const did = `did:btcr:${nameInfo.address}`
```

Comment:
This isn't the `did:stacks` DID method.  Also, in Stacks v2, the addresses will no longer be on the Bitcoin chain.

Reply:
btcr was easier to verify. But you are right, that did method shouldn't be part of this tool.